.ci.yaml

```diff
@@ -68,8 +70,13 @@ platform_properties:
         ]
       device_type: none
       mac_model: "Macmini8,1"
-      os: Mac-10.15
-      xcode: 12c33 # xcode 12.3
+      os: Mac-12
+      runtime_versions: >-
+        [
+          "ios-13",
```

Comment:
Some places, these are specified as ios-13-0. Not sure if it matters?

Reply:
In the past we were tracking minor versions just to be sure that updates were not breaking the builds and or generating different performance results.

Reply:
You were right, these need to match the cipd refs, `ios-13-0` etc
```
#0 mac_toolchain/install.go:311 - main.resolveRuntimeRef()
  reason: Failed to resolve runtime ref given runtime version: ios-13, xcode version: .
```
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8820126450450034161/+/u/install_xcode_runtime_ios-13/stdout